### PR TITLE
refactor: authorized keys message on add model

### PR DESCRIPTION
**Pull request by tlm**
In previous PR's we have removed the ability to add juju ssh keys when adding a model due to security impacts. This left an ugly error message users would see when adding models.

Now when adding a model we simply inform the user of what they need to do from there onwards.

## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- [x] Comments saying why design decisions were made
- ~[ ] Go unit tests, with comments saying what you're testing~
- ~[ ] [Integration tests](https://github.com/juju/juju/tree/main/tests), with comments saying what you're testing~
- ~[ ] [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages~

## QA steps

Bootstrap a controller and run `juju add-model foo` to check the new message.

## Documentation changes

N/A

### CODE REVIEW

**Review by wallyworld — APPROVED** [review]

**Review by hpidcock — APPROVED** [review]

File: cmd/juju/controller/addmodel.go [hunk 1]
```diff
@@ -322,14 +321,7 @@ func (c *addModelCommand) Run(ctx *cmd.Context) error {
...
 
-	if _, ok := attrs[config.AuthorizedKeysKey]; !ok {
-		// It is not an error to have no authorized-keys when adding a
-		// model, though this should never happen since we generate
-		// juju-specific SSH keys.
-		ctx.Infof(`
-No SSH authorized-keys were found. You must use "juju add-ssh-key"
-before "juju ssh", "juju scp", or "juju debug-hooks" will work.`)
-	}
+	ctx.Infof("To use \"juju ssh\", \"juju scp\" and \"juju debug-hooks\" ssh public keys need to be added to the model with \"juju add-ssh-key\"")
```
Comment (SimonRichardson) [inline cmd/juju/controller/addmodel.go:324]: I think the message is better, but I think the start is a bit off. Yet I don't have anything that comes to mind.

@tmihoc any clues, you're really good at this :point_up:

Comment (tmihoc) [reply cmd/juju/controller/addmodel.go:324]: @SimonRichardson What about the start is bothering you?

I don't know the context for this. (1) Are we anticipating a question a user in this context will actually have or (2) is this a random by-the-way? Without knowing that, all I can say is that 

(a) I don't like the "need to be added" -- I'd rephrase as "make sure to add".

(b) I don't like the "with \"juju add-ssh-key\"" bit -- it suggests you have to do it that way, and that's not true because you can also do this with `terraform-provider-juju` and `python-libjuju`, plus that info is really not-at-issue. I'd make it parenthetical:

> ... add your ssh public keys to the model (e.g., with `juju add-ssh-key`)

or, pretending like the other clients don't exist but still weakening the implications of the non-parenthetical "with":

> ... add your ssh public keys to the model (`juju add-ssh-key`)
